ONSdigital/response-operations-ui
# Fix front end bugs raised

# What and why?
As a result of testing from the DS upgrade a few issues have been raised:
- Surveys Secure Messaging Inboxes - input width increased to display full 11 digit RU ref
- When creating a SEFT collection exercise, we are unable to upload collection instrument, option to click upload is greyed out - update class name in js

Also:
- updates some buttons to use the `variants` param rather than `classes`
- fixes the button style for some buttons

# How to test?
- Check that filter input on Secure Messaging Inboxes is wide enough for 11 numbers
- Check that SEFT upload button is enabled when a .xls or .xlsx file is selected to upload
- Check that acceptance tests pass

# Trello
